src/markdown-preview-view/global-shims.d.ts

```diff
@@ -0,0 +1,5 @@
+interface MathJaxConfig {
```

Comment:
This looks like a duplicated definition. Not good. Defining this in `src-client/ipc.d.ts` in the `global` namespace would probably work better.